content/en/apps/guides/debugging/obtaining-logs.md

```diff
@@ -1,27 +1,63 @@
+weight:
 description: >
-  How to obtain logs from an instance
+  How to obtain server and client logs
 relatedContent: >
-  
+
 ---
 
 There are many places where useful logs reside. This details all those places, and the easiest way to get a hold of them.
 
-## Browser logs
+## Client logs
 
-To check if there are relevant logs open up the developer console. The shortcut is probably COMMAND+ALT+I on MacOS, or CTRL+ALT+I on Linux and Windows. Click the console tab and copy out any errors or logging that you think is relevant.
+### On a laptop or desktop
 
-## Server logs
+To check if there are relevant logs open up the developer console in your browser. The shortcut is probably COMMAND+ALT+I on MacOS, or CTRL+ALT+I on Linux and Windows. Click the console tab and copy out any errors or logging that you think is relevant.
+
+### On a phone
+
+There are two types of logs updated by Android devices depending on what information is needed. If you trying to get either of the logs from a device for the first time you need to set it up for USB debugging.
+
+1. Enable USB debugging on the phone. This varies from phone to phone, but here is the [official Android documentation](https://developer.android.com/studio/debug/dev-options#enable).
+2. Connect your phone by USB to the computer.
+3. You will see a popup _"Allow USB debugging. The computer's RSA fingerprint..."_, click _"OK"_. If you don't see the popup when the cable is plugged in, you will see it after the next step.
+
+   ![Allow USB debugging](/apps/guides/debugging/images/allow_usb_debugging.png)
+
+#### Browser logs
+
+These are the same as the desktop browser logs. Now that you've enabled USB debugging on the phone you can access the dev console on the phone via USB from your desktop browser. Follow the [official Android documentation](https://developer.chrome.com/docs/devtools/remote-debugging/webviews/#open_a_webview_in_devtools) to access the console.
+
+#### Android logs
 
-The easiest way is to use `cht-logs`, a tool that comes with [`cht-conf`](https://github.com/medic/cht-conf):
+The Android log is written to from the cht-android wrapper which can be useful to debug application crashes or failing integrations between apps.
 
+1. To **install** the `adb` command follow the instructions under the [Development Environment > Debug tool adb]({{< ref "core/guides/android/development-setup#debug-tool-adb" >}}) section.
+2. Within a command line session write the following command: `adb start-server`.
+3. To check if your phone is properly connected, write the command `adb devices`. This will list the devices connected.
+
+   ![ADB Devices](/apps/guides/debugging/images/adb_devices.png)
+
+4. Type the command `adb logcat > phone.log` to get the android logs in the file `phone.log`. You can stop the recording at any time pressing _Ctrl+C_. Now while taking android logs is the time to try to reproduce the error. Do this for as many times as you deem necessary to capture logs for that event.
```

Comment:
> Do this for as many times as you deem necessary to capture logs for that event.

Maybe call out that the log will get overwritten every time you run the command?

Reply:
That sentence is clunky. I think it's saying "reproduce the error" as many times as necessary, not run the command as many times as necessary. What about this...

```suggestion
4. Type the command `adb logcat > phone.log` to get the android logs in the file `phone.log`. The log will be written to the `phone.log` file as long as this command is running. You can stop the recording at any time pressing _Ctrl+C_. Now if you reproduce the error on the phone you can look for any useful information being written to the `phone.log`.
```

Reply:
yes, more better - thanks!